indexer/api/models/models.go

```diff
@@ -43,3 +48,38 @@ type WithdrawalResponse struct {
 	HasNextPage bool             `json:"hasNextPage"`
 	Items       []WithdrawalItem `json:"items"`
 }
+
+// FIXME make a pure function that returns a struct instead of newWithdrawalResponse
+// newWithdrawalResponse ... Converts a database.L2BridgeWithdrawalsResponse to an api.WithdrawalResponse
+func CreateWithdrawalResponse(withdrawals *database.L2BridgeWithdrawalsResponse) WithdrawalResponse {
+	items := make([]WithdrawalItem, len(withdrawals.Withdrawals))
+	for i, withdrawal := range withdrawals.Withdrawals {
+
+		msg := withdrawal.L2BridgeWithdrawal.CrossDomainMessageHash
+		if msg == nil {
+			msg = &common.Hash{}
```

Comment:
this sgtm, the 0x0 can be an indicator if the withdrawal bypassed the cross domain messenger